### PR TITLE
Removes shooooom sound from RPED/RPD

### PR DESCRIPTION
**Pull request by Kyep**
## What Does This PR Do
Removes the 3% chance that, when using a bluespace RPD or RPED, it will play this sound effect:
https://github.com/ParadiseSS13/Paradise/blob/master/sound/items/pshoom_2.ogg
instead of the normal ping or click sound effect.

## Why It's Good For The Game
1. The pshoom_2.ogg sound effect is jarringly out of place in the game. It sounds like a person saying "shooom", not like any actual sound a machine would ever make. It is so out of place that the first few times I heard it, I thought some kind of ad must have snuck into one of my browser tabs. I did not even recognize it as an in-game sound at all.
2. The fact that it only plays at a 3% chance, means that you'll rarely hear it, makes it even more confusing, as you can't reproduce it reliably, so its hard to find out what exactly causes it. Most of the people hearing it won't be the person using the RPED/RPD, so they're unlikely to know what it is except by learning the RNG over time. Overall, it is confusing and unintuitive.
3. The code to support it seems a bit snowflakey. Unlike most items that use the standard 'usesound' var, the code here still has 'usesound' but then ignores it, instead randomly switching between 'primary_sound' and 'alt_sound' which only exist on these types.

## Changelog
:cl: Kyep
del: Bluespace RPEDs/RPDs no longer have a 3% chance to play a 'shoooooom' sound on use.
/:cl:

### CODE REVIEW

**Comment by Ty-Omaha** [comment]
This is like the best sound in the game

**Comment by Ty-Omaha** [comment]
OH. The weird guy saying "SHOOOOOM" not the sound effect when you use it.

**Comment by Pixelzilla** [comment]
This is literally my favorite sound in the game. Its funny. It doesn't harm anything and I don't really see why its an issue and if it is such a low chance why does it really matter? Sometimes it is nice to have silly things in ss13. I just don't like that a lot of these silly/quirky/fun/wacky things that make ss13 are slowly being purged. We're gonna end up with a boring job simulator at this rate.

**Comment by Fox-McCloud** [comment]
Yeahhh, no.

This has been around for literally years now--it's hilarious, doesn't really break immersion as much as it's made out to be, and is a neat little easter egg that really doesn't happen all that often, not to mention this fits perfectly with the atmosphere of SS13.

SS13 is whacky, silly, and not 200% serious---not to mention people do silly stuff like this, all the time, in real workplaces, just to get a cheap laugh, so it's not that immersion breaking or unrealistic.

Huge 👎

**Comment by Kyep** [comment]
Funny, wacky, silly, etc are all fine.

Thing is, stuff still needs to fit within the game, and this doesn't.

If it was a goofy series of beeps, or even something like a bike horn, it would still be wacky/silly/goofy/etc but it would at least be recognizable as a SS13 sound.
As it is, it is the sort of sound that makes you check your browser tabs to see if there's an ad playing somewhere, or someone is talking to you in a voice channel.
In other words: it doesn't sound like an ingame sound effect, at all. It is more confusing than funny.

**Comment by Fox-McCloud** [comment]
This has existed since prior to the vast majority of our current players here have existed. Literally no one but you has ever commented on how jarring it is or strange it is---it's been almost universally celebrated as hilarious and giggle-worthy, and a good bit of typical SS13 fun.

It's not as jarring as you're making it out to be, nor is it that jarring in terms of where it's coming from or why---and people will even sometimes deliberately interject this themselves, because of how hilarious it is.

At this stage, this is being removed for little reason--for something that, I'll be honest, I am boggled you haven't noticed after 5 years, until now.

**Comment by Kyep** [comment]
Something being old is not an automatic reason to keep it. Practically, we remove old stuff regularly. We're replacing a bunch of nanoui as I speak. More generally in principle, old is not automatically good.

You're incorrect about nobody commenting on how strange/jarring it is. I've had a couple of people mention to me in voice today about how it sounds weird.

It isn't universally celebrated as hilarious or giggle-worthy either. Actually, outside of the comments on this PR, nobody I've asked about it has told me they find it funny. And I've brought it up a few times. Prior to creating this PR, I wasn't aware there was anyone who liked this sound effect.

I noticed this sound effect soon after I started playing on the server. It took me awhile to figure out what was causing it. When I did, I considered it dumb, but since I was new at the time, I did not propose changing it. Over the years, every time it has come up, I felt it was out of place. I've felt like it was out-of-place for years. But today just happened to be the day I decided to stop being annoyed with it every time it came up, and actually do something about it.

Fun, silly, wacky, etc is fine. But this sound effect is so out-of-place it makes people tab out of the game and check their browser windows because it doesn't seem to be a game sound effect.

**Comment by MRgaty** [comment]
Is a joke sound that plays rarely, is silly and I love it.
I dont see why having it could be bad for the game, given it is some sort of "easter egg".
I would lower the chance of it playing tho.

**Comment by Spacemanspark** [comment]
It doesn't help matters that the sound file often has a short delay, making it play over the regular sound effect and stirring confusion if you're quickly upgrading things.

Admittedly, the sound was funny the first few times, but it lost its charm rather quickly for me. At the very least, it needs to have its chance of playing lowered by a significant amount imo.

**Comment by Fox-McCloud** [comment]
>Something being old is not an automatic reason to keep it. Practically, we remove old stuff regularly. We're replacing a bunch of nanoui as I speak. More generally in principle, old is not automatically good.

We get rid of old things like this because something better comes along to replace them, they cause performance issues, or they're dramatically underutilized, or they become abusive due to a shift in overall game design--things that have generally been around a long time and not been complained about (ie: this) can much more easily stand on tradition--especially once you get past the 2-3 year mark where they start becoming a part of SS13 culture.

>You're incorrect about nobody commenting on how strange/jarring it is. I've had a couple of people mention to me in voice today about how it sounds weird.

It isn't universally celebrated as hilarious or giggle-worthy either. Actually, outside of the comments on this PR, nobody I've asked about it has told me they find it funny. And I've brought it up a few times. Prior to creating this PR, I wasn't aware there was anyone who liked this sound effect.

If you ask around enough, you'll find someone to match your opinion; while I don't doubt that there's a couple people out there---having been around this server for longer than just about anyone else still around---I've yet to hear of this being clamored for removal or this being that jarring.

> If the alternate sound was something that you could imagine coming out of a machine, even if it was very comic, it would be fine.

It's quite clear it's the person making the sound as opposed to the machine. Again, people in real workplace environments act silly like this to make mundane things at least tentatively more amusing/less ...mundane, all the freaking time.

**Comment by Spacemanspark** [comment]
A) What if my character is female
B) What if my character isn't the kind to make such a sound
C) Why is my character's voice being assumed, that's voiceist
(yes, C is a joke)

**Comment by Dumbdumn5** [comment]
To repeat what I put in headchat when asked about approval or disapproval for this PR:



> It's an easter egg, a joke noise put in for no other reason than levity. If it didn't land for you, it's perfectly fine, but individual personal opinion should not dictate what should or should not be within the sense of acceptable 'gameiness' or 'wackiness' when it's more or less just a community meme. I'd argue that there should be room for things such as this to take people out of it for a moment, maybe chuckle and wink at it or roll their eyes and scoff in the case of some, to give people the sense that, regardless of how seriously it could be taken, it's still a game, and the minor and relatively rare immersion break is a good way of doing that non-intrusively.

> Collective personal opinion, in this case, seems to be against the idea that this is unacceptable, looking at the PR's current responses, and I am more inclined to respect the enjoyment of the community than I am inclined to put my 'no fun allowed' boot on something that's arguably a non-issue such as this. 

The github isn't exactly the best sample size since fewer people use the git than the forums, likely, which means that this is often feedback from just a handful of people, but considering we don't have many alternatives, this is the best thing I've got to gauge how people feel about it. As such, due to this PR being primarily based on personal aesthetic opinion and nothing drastically game-changing, I'm going with the majority community opinion based on what feedback we have available here.

:-1:

**Comment by Carthusia** [comment]
Personally i find it hilarious,i just picture a drask using it saying shoooooooooooooooooooooooom as they work
